libs/common/include/launchdarkly/bindings/c/config/logging_builder.h

```diff
@@ -0,0 +1,134 @@
+/** @file */
+// NOLINTBEGIN modernize-use-using
```

Comment:
This was extracted out of `sdk.h`. It can remain in common because it'll be shared by the client and server.